webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/queries/DynGroupService.scala

```diff
@@ -153,7 +153,7 @@ class DynGroupServiceImpl(
   override def changesSince(lastTime: DateTime):   Box[Boolean]                              = {
     val n0 = System.currentTimeMillis
     if (n0 - lastTime.getMillis < 100) {
-      Full(true)
+      Full(false)
```

Comment:
here, if I understand correctly the code and how it's used in `processUpdate` (`val need = dynGroupService.changesSince(lastUpdateTime).getOrElse(true)`), `true` means that we DO need to compute groups. But if last change is not far away, we likely want to avoid that.